src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1661,7 +1672,6 @@ public void testCachedFile() throws Exception {
         checker.destroy();
     }
 
-    @SuppressForbidden
```

Comment:
Please keep it, we are still violating rule to not use deprecated method, and it shows us that we can not avoid usage of it all, until we support Checker in inlined config.

Please put this annotation on method that you created.

Reply:
Done